scripts/postinstall.sh

```diff
@@ -20,3 +22,7 @@ rm -rf /root/.ssh /root/.packer_http
 # Zero out the disk (for better compression)
 dd if=/dev/zero of=/EMPTY bs=1M
 rm -rf /EMPTY
+
+echo "Call nixos-enter"
+
+nixos-enter
```

Comment:
I'm not sure this is how it should be used. The whole script should execute inside of the nixos-enter context, otherwise the commands are going to be executed on the ISO nixos